playbooks/openshift-node/private/join.yml

```diff
@@ -36,54 +62,46 @@
 
   - name: Distribute bootstrap kubeconfig if one does not exist
     copy:
-      content: "{{ bootstrap_credentials.content | b64decode }}"
+      src: "{{ hostvars['localhost'].mktemp.stdout }}/bootstrap.kubeconfig"
       dest: "/etc/origin/node/bootstrap.kubeconfig"
       mode: 0700
       owner: root
       group: root
       force: no
     no_log: true
 
-- name: Approve any pending CSR requests from inventory nodes
-  hosts: oo_first_master
-  gather_facts: no
-  tasks:
-  - name: Dump all candidate bootstrap hostnames
-    debug:
-      msg: "{{ groups['oo_nodes_to_bootstrap'] | default([]) }}"
-
-  - name: Find all node names
-    set_fact:
-      l_nodes_to_join: "{{ groups['oo_nodes_to_bootstrap'] | default([]) | map('extract', hostvars) | map(attribute='openshift.node.nodename') | list }}"
-
-  - name: Dump the bootstrap hostnames
-    debug:
-      msg: "{{ l_nodes_to_join }}"
+  - name: Ensure CNI config is created
```

Comment:
No, this should not be done in ansible.  This is owned by the sdn plugin.  If an extender needs to do this it needs to be part of the component step.

Reply:
Sure, I'll check if it works without it